jeremy2918/tsseract
# Updated Scripts

# [Updated Scripts](https://trello.com/c/3yzyXdSo/32-i-want-to-update-the-project-scripts)

## Description
Updates npm scripts


## Testing Steps
- `npm install`
- `npm run watch` should start both server and client on watch mode
- `npm run tsc` should start TypeScript compiler in watch mode.



## Changes
No UI Changes
